File: framework/src/main/java/com/software/dm/swallow/stormy/algoac/pojo/Param.java
package com.software.dm.swallow.stormy.algoac.pojo;

/**
 * @author DM
 * @version v1.0.0.1
 * @Description
 * @date 2017
 * @preserve
 */
public final class Param {


    private int type = 0;// ???????
    private String text = "";
    private Object obj = null;


    /**
     * @return
     * @preserve
     */
    public int getType() {
        return type;
    }

    public String getText() {
        return text;
    }

    public Object getObj() {
        return obj;
    }

    public Param(int type, String text, Object obj) {
        this.type = type;
        this.text = text;
        this.obj = obj;
    }

    @Override
    public String toString() {
        return "Param [type=" + type + ", text=" + text + ", obj=" + obj + "]";
    }

}

File: framework/src/main/scala/com/software/dm/swallow/stormy/scala/algoac/pojo/Param.scala
package com.software.dm.swallow.stormy.scala
package algoac
package pojo


/**
  * @author darlingming@126.com
  * @param `type`
  * @param text
  * @param obj
  */
final class Param(val `type`: Int = 0, val text: String, val obj: Any) {



  def getType: Int = `type`

  def getText: String = text

  def getObj: Any = obj

  override def toString: String = "Param [type=" + `type` + ", text=" + text + ", obj=" + obj + "]"
}

